runtime_lib/CMakeLists.txt

```diff
@@ -109,6 +113,9 @@ foreach(target ${AIE_RUNTIME_TARGETS})
           -DVITIS_ROOT=${VITIS_ROOT}
           -DVITIS_AIETOOLS_DIR=${VITIS_AIETOOLS_DIR}
           -DSysroot=${Sysroot}
+          -DMLIR_AIR_DIR=${MLIR_AIR_DIR}
```

Comment:
do we really depend on mlir-air here? seems a bit circular

Reply:
Good catch, we don't depend on mlir-air this is an artifact of a previous version where we did. Will remove this.

Reply:
Thanks, addressed in [f18a8d6](https://github.com/Xilinx/mlir-aie/pull/570/commits/f18a8d66b255ed857b56380a62fe27e5ba046e25).